vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -44,29 +45,34 @@ def _get_id(site: dict) -> str:
     return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
 
 
+def _cleanup_url(url):
+    if not url:
+        return None
+    # if "@" in url:
+    #     # Some of these are email addresses.
+    #     # Skipping those for now.
+    #     return None
+    if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
```

Comment:
Perhaps a more generic rule would be to strip leading and trailing spaces, and then skip entries that still have spaces (since the can't be a url)

Reply:
good catch! Do you think this would be useful to make into a shared utility function for cleaning up urls?

Reply:
Yes

Reply:
aight will do